.docker/aiida-core-base/s6-assets/init/aiida-prepare.sh

```diff
@@ -5,24 +5,21 @@
 # Environment.
 export SHELL=/bin/bash
 
-# Configure AiiDA.
-export SETUP_DEFAULT_AIIDA_PROFILE=true
-export AIIDA_PROFILE_NAME=default
-export AIIDA_USER_EMAIL=aiida@localhost
-export AIIDA_USER_FIRST_NAME=Giuseppe
-export AIIDA_USER_LAST_NAME=Verdi
-export AIIDA_USER_INSTITUTION=Khedivial
-export AIIDA_PROFILE_PATH=/aiida/assets/config-quick-setup.yaml
+# If the environment variable `SETUP_DEFAULT_AIIDA_PROFILE` is not set, set it to `true`.
+if [ -z ${SETUP_DEFAULT_AIIDA_PROFILE+x} ]; then
+    # Configure AiiDA.
+    export SETUP_DEFAULT_AIIDA_PROFILE=true;
+fi
 
 # Check if user requested to set up AiiDA profile (and if it exists already)
 if [[ ${SETUP_DEFAULT_AIIDA_PROFILE} == true ]] && ! verdi profile show ${AIIDA_PROFILE_NAME} &> /dev/null; then
-    NEED_SETUP_PROFILE=true;
-else
-    NEED_SETUP_PROFILE=false;
-fi
 
-# Setup AiiDA profile if needed.
-if [[ ${NEED_SETUP_PROFILE} == true ]]; then
+    export AIIDA_PROFILE_NAME=default
+    export AIIDA_USER_EMAIL=aiida@localhost
+    export AIIDA_USER_FIRST_NAME=Giuseppe
+    export AIIDA_USER_LAST_NAME=Verdi
+    export AIIDA_USER_INSTITUTION=Khedivial
+    export AIIDA_PROFILE_PATH=/aiida/assets/config-quick-setup.yaml
```

Comment:
Should these also maybe just set these defaults only if not already explicitly set?
e.g.
```
export AIIDA_PROFILE_NAME="${AIIDA_PROFILE_NAME:-default}"
```

Reply:
Also, maybe update the docs to reflect that these variables can be configured

Reply:
Yes, this looks much simpler, and clear.